sysconfigs/daint/packages.yaml

```diff
@@ -35,7 +35,7 @@ packages:
       compiler: [pgi]
     gridtools:
       variants: +cuda cuda_arch=60
-      compiler: [gcc]
+      compiler: [gcc@8.3.0]
     cosmo-dycore:
       variants: slave=daint +cuda cuda_arch=60 data_path=/scratch/snx3000/jenkins/data/cosmo/ slurm_args= "'-C gpu -p normal -A g110 -N {0}'"
       compiler: [gcc]
```

Comment:
we don't need the 8.3.0 here ? Is this set by the fact that it depends on gridtools (just asking because gridtools is a header only library, so the actual compilation occurs in cosmo-dycore)

Reply:
You are right, this should be on line 41 for the cosmo-dycore